### DISCUSSION TITLE
Leerfahren durch Umpolung der Pumpen

### DISCUSSION THREAD

**Discussion by Marvin0795** [AndreWohnsland/CocktailBerry#147] [Ideas]
![Screenshot 2024-02-13 103431](https://github.com/AndreWohnsland/CocktailBerry/assets/102437306/97bc74e7-9a73-4f75-8891-e016af50a1db)

Könnte es Sinnvoll sein, nach Aktivierung einer Umschaltung der Richtung in den Einstellungen, ein weiteren Button zu haben, der die Anlage Leer fährt und wir dann Reinigen können?

**Comment by AndreWohnsland** [comment]
Es gibt ja schon das Reinigungsprogramm, wenn die Reversion aktiviert ist, wird der Pin vor der Reinigung aktiviert (wenn der user ja drüclt), was die Stromflussrichtung umkehrt. Somit wird die Pumpe während der Reinigung dann andersherum gefahren. Ist es das, was du gemeint hast, oder ist das was anderes?

![image](https://github.com/AndreWohnsland/CocktailBerry/assets/50302161/d9cf1298-14a1-450d-bf67-f9458b04fb4f)

**Reply by Marvin0795** [reply]
Ah perfekt, die Funktion war mir so noch nicht bekannt, dass reicht mir schon. 
Also wenn das Reinigungsprogramm gestartet wird, dass er dann automatisch Abfragt, wenn der Pin unter einstellung aktiviert ist und dann rückspült. 
Vielleicht kannst du in der Doku unter dem Reinigungsprogramm erwähnen, dass, falls angehakt, die pumpen umgekehrt werden . :-)

**Reply by AndreWohnsland** [reply]
Ja, das macht Sinn das dort noch zu ergänzen. Wenn die Option an ist, kommt das Fenster vor dem Reinigen und man kann die Richtung umkehren, wenn man ja klickt. Ich selbst nutze das nicht, da ich Membran Pumpen habe, aber Leute mit Schlauchlumpen können das zum Beispiel nutzen.

**Reply by Marvin0795** [reply]
Wir verwenden auch Membran Pumpen, hatten aber bei 1-2 Flüssigkeiten über eine Schlauchpumpe nachgedacht
